### PR TITLE
Explicitly support ipv4 in invoke

### PR DESCRIPTION
**Pull request by rylandg**
Ipv6 is causing issues for people.

### CODE REVIEW

**Comment by rylandg** [comment]
Sorry that this was half-assed, I was rushing to get it done. I’ll update
for deploy right away. I can’t post context because it’s customer
sensitive, but I’ll message you on slack.

**Comment by michaeladda** [comment]
@rylandg any update?

File: sdk/auth.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ const verifyAPIKey = async function verifyAPIKey(apiKey) {
       headers: {
         'X-Binaris-Api-Key': apiKey,
       },
+      family: 4,
```
Comment (vogre) [inline sdk/auth.js]: I think no reason to use raw request here,  possible to use `getValidatedBody` to get all the benefits from reuse?

**Comment by rylandg** [comment]
It did pass but doesn't look like it from here. Merging.

**Pull request merged**